Failing build as of 5.4.0-rc2

Hello, this commit in 5.4.0-rc2 renamed `nf_reset` to `nf_reset_ct`, which broke the build:

* https://github.com/torvalds/linux/commit/895b5c9f206eb7d25dc1360a8ccfc5958895eb89

Failure looks roughly like this:

```
[03:15:15]	make -C /huh/build/linux-5.4.0-rc2 ARCH=x86_64  O=/huh/build/amd64 M=/huh/build/glb-director/src/glb-redirect modules
[03:15:15]	make[1]: Entering directory '/huh/build/linux-5.4.0-rc2'
[03:15:15]	make[2]: Entering directory '/huh/build/amd64'
[03:15:16]	grep: dkms.conf: No such file or directory
[03:15:16]	  CC [M]  /huh/build/glb-director/src/glb-redirect/ipt_GLBREDIRECT.o
[03:15:17]	/huh/build/glb-director/src/glb-redirect/ipt_GLBREDIRECT.c: In function 'glbredirect_send_forwarded_skb':
[03:15:17]	/huh/build/glb-director/src/glb-redirect/ipt_GLBREDIRECT.c:94:2: error: implicit declaration of function 'nf_reset'; did you mean 'nf_ct_set'? [-Werror=implicit-function-declaration]
[03:15:17]	   94 |  nf_reset(skb);
[03:15:17]	      |  ^~~~~~~~
[03:15:17]	      |  nf_ct_set
[03:15:17]	cc1: some warnings being treated as errors
[03:15:17]	make[2]: Leaving directory '/huh/build/amd64'
[03:15:17]	make[3]: *** [/huh/build/linux-5.4.0-rc2/scripts/Makefile.build:266: /huh/build/glb-director/src/glb-redirect/ipt_GLBREDIRECT.o] Error 1
[03:15:17]	make[1]: Leaving directory '/huh/build/linux-5.4.0-rc2'
[03:15:17]	make[2]: *** [/huh/build/linux-5.4.0-rc2/Makefile:1650: /huh/build/glb-director/src/glb-redirect] Error 2
[03:15:17]	make[1]: *** [Makefile:179: sub-make] Error 2
[03:15:17]	make: *** [Makefile:195: build-glb-redirect-amd64] Error 2
[03:15:19]	Failure: 2
```

--- a/src/glb-redirect/ipt_GLBREDIRECT.c
+++ b/src/glb-redirect/ipt_GLBREDIRECT.c
@@ -91,7 +91,11 @@ static unsigned int is_valid_locally(struct net *net, struct sk_buff *skb, int i
 static unsigned int glbredirect_send_forwarded_skb(struct net *net, struct sk_buff *skb)
 {
 	struct glbgue_stats *s = this_cpu_ptr(percpu_stats);
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,4,0)
+	nf_reset_ct(skb);
+#else
 	nf_reset(skb);
+#endif
 	skb_forward_csum(skb);
 
 	if (ip_route_me_harder(

